ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/private/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
Copied from https://github.com/openshift/release/pull/24501
> I only use soft-links if I'm in the parent OWNERS, and there're updates done by me in the current folder.
The same suggestion as before, to avoid affect from the potential OWNERS change of the parent folder, and keep the same style as other platforms.